ttnn/cpp/ttnn/operations/data_movement/non_zero_indices/non_zero_indices_pybind.hpp

```diff
@@ -0,0 +1,55 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+#include "ttnn/cpp/pybind11/decorators.hpp"
+
+#include "non_zero_indices.hpp"
```

Comment:
please split to hpp and cpp
best example is ccl/all_gather
it also shows how to use pybind_fwd.hpp to not include anything heavy in this header

Reply:
done